src/runners/rfq_maker_balance_cache_runner.ts

```diff
@@ -80,7 +80,10 @@ if (require.main === module) {
         const balanceCheckerContractInterface = getBalanceCheckerContractInterface(RANDOM_ADDRESS, provider);
 
         await runRfqBalanceCacheAsync(web3Wrapper, connection, balanceCheckerContractInterface);
-    })().catch(error => logger.error(error.stack));
+    })().catch(error => {
+        logger.error(error.stack);
+        process.exit(1);
```

Comment:
This is great!! This will cause the process to fail and it will force K8 to create a new instance.

Reply:
👍